Tools/Postprocessing/C_Src/Make.package

```diff
@@ -1,6 +1,7 @@
 CEXE_sources += ${EBASE}.cpp
 
-ifneq ($(EBASE), particle_compare)
+ifeq ($(EBASE), PtwisePltTransform)
+ifeq ($(EBASE), PlotfileToTurb)
```

Comment:
No EBASE will pass both tests.  So it's always false.

Reply:
Doh!  Guess we need the previous logic then of nested ifneq's?

Reply:
yeah, it looks like some but not all of the other EBASE options do need amrdata, which is the main contribution of that part of the loop (like `PlotfileToMatLab`). My original suggestion was wrong for that reason too.

Reply:
What's pushed at the moment works with all EBASE options.